Operational-Procedures/Troubleshooting/component-offline.rst

```diff
@@ -0,0 +1,148 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-component-offline:
+
+#################
+Component Offline
+#################
+
+.. warning::
+
+    This procedure contains steps that may affect the system in crical ways.
+
+    If you are unsure about any of the steps described here, contact support personnel.
+
+
+* Most (but not all) components in the Rubin Observatory Control System, when sent to OFFLINE will terminate its execution process and will no longer be available in the system.
+
+* This condition only happens if someone sends the ``exitControl`` command to the CSC.
+
+  Sending the ``exitControl`` command to a CSC is regularly done when one wants to update a version of the CSC or needs to restart the process to recover from some fault conditions.
+
+* A reduced number of CSCs will continue to run after receiving ``exitControl``.
+
+  At the time of this writting, only the camera systems have this behavior.
+
+Restoring a CSC that exited after responding to ``exitControl`` requires one to restart the process that runs it.
```

Comment:
Change to: "Restoring a CSC after sending an ``exitControl`` requires one to restart the process that runs it.